specification/dataprotection/resource-manager/readme.md

```diff
@@ -46,6 +46,15 @@ semantic-validator: true
 message-format: json
 ```
 
+### Tag: package-2021-10-preview
+
+These settings apply only when `--tag=package-2021-10-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-10-preview'
```

Comment:
seems the default tag is still 'tag: package-2021-07', is this expected?

Reply:
its --tag=package-2021-10-preview; please refer to latest iteration on this file.